bin/lib/get_cmd_args.sh

```diff
@@ -16,13 +16,16 @@ function get_cmd_args::verify_config_file_exists() {
   fi
 }
 
-function get_cmd_args::get_config_file() {
+function get_cmd_args::get_args() {
   for i in "$@"; do
     case "${i}" in
       --config=*)
         export CONFIG="${i#*=}"
         get_cmd_args::verify_config_file_exists
-        return
+        ;;
+      --cert=*) 
+        export CERT_FILE="${i#*=}"
```

Comment:
(blocking) If this flag is set we should check that the file exists and is readable before continuing. Something along these lines should work.

```bash
if [[ -z "${CERT_FILE}" || ! -r "${CERT_FILE}" ]]; then
  echo "some message"
  exit 1 # some appropriate code
fi
```

(question) Is there a certain certificate file format this need to be? If yes, we should try to test it's the type we need and exit early if not.

Reply:
Yes, good call! I added this in another function to follow the convention of the config file check. Going to work with Charlotte to figure out the cert file format and update to check that as a follow up.